includes/Blocks/Block.php

```diff
@@ -122,65 +123,184 @@ private function register_block_attributes_as_fields(): void {
 	}
 
 	/**
-	 * Gets the WPGraphQL field registration config for the block attributes.
-	 *
-	 * @param ?array $block_attributes The block attributes.
+	 * Returns the type of the block attribute
+	 * 
+	 * @param string $name The block name
+	 * @param object $attribute The block attribute config
+	 * @param string $prefix Current prefix string to use for the get_query_type
 	 */
-	private function get_block_attribute_fields( ?array $block_attributes ): array {
-		$block_attribute_fields = [];
-
-		// Bail early if no attributes are defined.
-		if ( null === $block_attributes ) {
-			return $block_attribute_fields;
-		}
-
-		foreach ( $block_attributes as $attribute_name => $attribute_config ) {
-			$graphql_type = null;
-
-			if ( ! isset( $attribute_config['type'] ) ) {
-				return $block_attribute_fields;
-			}
+	private function get_attribute_type( $name, $attribute, $prefix ): mixed {
```

Comment:
I believe `mixed` is only available in PHP 8.0+. The plugin currently works with PHP 7.4, so this would fatal error on sites using that. Not opposed to bumping min requirements of the plugin, but that's something we should discuss first.

Reply:
I don't see a need to bump from 7.4 yet.

Like with union returns, if we _really need to use a mixed_, we can PHPDoc it. I'm on my phone and struggling to review the whole PR, but in this case im betting the actual type can be derived from the method itself.